### ISSUE TITLE
Machine Reboot on Train

### ISSUE THREAD

**Issue by adreamtraveler** [deepfakes/faceswap-playground#151]
Hi, I'm having an issues with machine reboots while faceswap starts the training process.

## Expected behavior

I have have successfully installed faceswap and extracted 64 test images for each face A and face B using "python faceswap.py extract".  I am now passing these to the train script to try and begin training the test model using "python faceswap.py train".

## Actual behavior

Using 'python faceswap.py train -A <path> -B <path> -m <path> -p -v -s 100 -t "Original" -bs 32 -e 10000000 -g '", I am now trying to begin the training process.  The process begins initialization and then my machine dies and reboots either just before or just after the training process begins.  Sometimes the training preview opens and a couple of loss values are printed and sometimes it dies before even printing "Starting" to the shell.  It's different each time, but not by much.

## Steps to reproduce

Run 'python faceswap.py train -A <pathA> -B <pathB> -m <model_path> -p -v -s 100 -t "Original" -bs 32 -e 10000000 -g 1' using two folders of extracted images produced with 'python faceswap.py extract'

## Other relevant information
- **Command lined used (if not specified in steps to reproduce)**: python faceswap.py train
- **Operating system and version:**: Windows7
- **Python version:**: 3.6.5
- **Faceswap version:**: 9b58b35
- **Faceswap method:**: GPU
- **Other related issues:**: None at this time
- **Hardware**:  gtx1080Ti 12gb, CUDA9, cuDNN7.0.5
- **pip freeze**:
absl-py==0.2.0
astor==0.6.2
bleach==1.5.0
click==6.7
cmake==3.11.0
cycler==0.10.0
decorator==4.3.0
dlib==19.10.0
face-recognition==1.2.2
face-recognition-models==0.3.0
gast==0.2.0
grpcio==1.11.0
h5py==2.7.1
html5lib==0.9999999
Keras==2.1.2
kiwisolver==1.0.1
Markdown==2.6.11
matplotlib==2.2.2
networkx==2.1
numpy==1.14.2
opencv-python==3.3.0.10
pathlib==1.0.1
Pillow==5.1.0
protobuf==3.5.2.post1
pyparsing==2.2.0
python-dateutil==2.7.2
pytz==2018.4
PyWavelets==0.5.2
PyYAML==3.12
scandir==1.6
scikit-image==0.13.1
scipy==1.0.1
six==1.11.0
tensorflow-gpu==1.5.0
tensorflow-tensorboard==1.5.1
termcolor==1.1.0
tqdm==4.19.6
virtualenv==15.2.0
virtualenvwrapper-win==1.2.5
webencodings==0.5.1
Werkzeug==0.14.1

Thanks

**Comment by bryanlyon** [comment]
This is a hardware problem with your system.  Either overheating, or undervolting.  There is nothing that we can do to avoid or fix this problem.  Check to make sure you have sufficient cooling and power for your graphics card or try another system.

**Comment by rharrison09** [comment]
I had this exact same problem when I built a PC that did not supply enough power after a gpu upgrade.

**Comment by adreamtraveler** [comment]
Thank you very much for your responses!  I have an 850w PSU, but also a very large mainboard running quad Xeon x5660 chips.  I am not overclocking any of the chips.  I'll run the stress tests you recommended and check the power connections to the card.  If things look ok there, then I'll see about a larger PSU and new wiring harness to test with.  Thanks again
